src/ResourceManager/Cdn/Commands.Cdn/help/New-AzureRmCdnProfile.md

```diff
@@ -157,9 +157,6 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## INPUTS
 
-### None
```

Comment:
@prakharsharma10 please revert the change made here

Reply:
This file is auto-generated using the platyPS module